content/ja/docs/contribute/style/hugo-shortcodes/index.md

```diff
@@ -0,0 +1,345 @@
+---
+title: カスタムHugoショートコード
+content_type: concept
+---
+
+<!-- overview -->
+このページではKubernetesのマークダウンドキュメント内で使用できるHugoショートコードについて説明します。
+
+ショートコードについての詳細は[Hugoのドキュメント](https://gohugo.io/content-management/shortcodes)を読んでください。
+
+<!-- body -->
+
+## 機能の状態
+
+このサイトのマークダウンページ(`.md`ファイル)内では、説明されている機能のバージョンや状態を表示するためにショートコードを使用することができます。
+
+### 機能の状態のデモ
+
+最新のKubernetesバージョンで機能をstableとして表示するためのデモスニペットを次に示します。
+
+```
+{{</* feature-state state="stable" */>}}
+```
+
+これは次の様に表示されます:
+
+{{< feature-state state="stable" >}}
+
+`state`の値として妥当な値は次のいずれかです:
+
+* alpha
+* beta
+* deprecated
+* stable
+
+### 機能の状態コード
+
+表示されるKubernetesのバージョンのデフォルトはそのページのデフォルトまたはサイトのデフォルトです。
+`for_k8s_version`パラメータを渡すことにより、機能の状態バージョンを変更することができます。
+例えば:
+
+```
+{{</* feature-state for_k8s_version="v1.10" state="beta" */>}}
+```
+
+これは次の様に表示されます:
+
+{{< feature-state for_k8s_version="v1.10" state="beta" >}}
+
+## 用語集
+
+用語集に関連するショートコードとして、`glossary_tooltip`と`glossary_definition`の二つがあります。
+
+コンテンツを自動的に更新し、[用語集](/docs/reference/glossary/)へのリンクを付与する挿入を使用して、用語を参照することができます。
```

Comment:
Japanese version exists.
(And there might be more links that can be updated to Japanese versions. I did not checked all.)

```suggestion
コンテンツを自動的に更新し、[用語集](/ja/docs/reference/glossary/)へのリンクを付与する挿入を使用して、用語を参照することができます。
```